### PR TITLE
Changed PoweredBy param to include a link 

### PR DESCRIPTION
**Pull request by SriHV**
### What is the context of this PR?

#3164...  Previously, there was no option to include a link around poweredBy logo when custom logo was used. Changed poweredBy param to include this.

### How to review this PR
Check that poweredBy param has two parameters `logoImage` and `logoUrl`. 
Test example-footer-with-alternative-organisation to ensure these parameters are shown

### Checklist

This needs to be completed by the person raising the PR.

- [x] I have selected the correct Assignee
- [x] I have linked the correct Issue

### CODE REVIEW

**Comment by balibirchlee** [comment]
Not sure how this fits together with the changes Precious made to the  poweredby functionality recently - I think we'll need to discuss in the next tech session.

**Review by rmccar — CHANGES_REQUESTED** [review]
I think some of these issues (at least the focus state one) might have been solved in the other footer PR. You can have a look at that https://github.com/ONSdigital/design-system/pull/3153

**Review by alessioventuriniAND — COMMENTED** [review]
It all looks good. Can you please fix the focus state that Richard has mentioned? 

For reference , more work will be completed on the footer logo on this issue. #3134 

This changes can still be applied as the afore mentioned issue is currently being re-worked/expanded on as the original requirements have changed therefore it is not longer a blocker for this issue. :)

File: src/components/footer/_footer.scss [hunk 1]
```diff
@@ -30,13 +30,13 @@
...
-                @extend %a-focus;
-            }
-        }
+  &__poweredBy-link {
+    &:focus {
+      .ons-icon--logo {
+        background-color: var(--ons-color-focus);
+      }
     }
+  }
```
Comment (alessioventuriniAND) [inline src/components/footer/_footer.scss:39]: Looks like this change is removing the underline to the default logo. 
Screenshot 2024-06-04 at 14 39 31

**Comment by SriHV** [comment]
This change is already being addressed in this #3134 PR. Closing this ticket

**Pull request closed without merging**